### PR TITLE
Go back to using AppVeyor to test windows

### PR DESCRIPTION
**Pull request by oxinabox**
Partially  reverts: https://github.com/invenia/Intervals.jl/pull/79

Need to do this because of:
https://github.com/JuliaIO/FFMPEG.jl/issues/14

This is so can test  #82

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+environment:
+  matrix:
+  - julia_version: 1.0  # latest LTS
+  - julia_version: 1  # latest stable
+  - julia_version: nightly
+
+platform:
+  - x86 # 32-bit
```
Comment (omus) [inline appveyor.yml]: Double testing on 32-bit again. I would just drop it from AppVeyor and do Linux 32-bit on Travis

**Comment by omus** [comment]
AppVeyor needs to be re-enabled. Also, Travis didn't run?

File: appveyor.yml (hunk 1 above)
Comment (oxinabox) [reply appveyor.yml]: Ah this time is is because I started from scratch in with AV
```suggestion
```

**Comment by oxinabox** [comment]
> AppVeyor needs to be re-enabled. 

It doesn't look to me like it was ever disabled
https://ci.appveyor.com/project/invenia/intervals-jl

> Also, Travis didn't run?

weird, it's running now

**Comment by oxinabox** [comment]
The trick used for TimeZones to not do plotting at all will not work for Intervals,
as we actually want to check what the plots look like in our tests,
not the attributes of them.

**Comment by oxinabox** [comment]
What things should I do to complete this?
For some reasonm Travis is still showing pending, but I think it is done when I click on details.

**Pull request merged**